src/index.js

```diff
@@ -200,6 +204,72 @@ class IpfsRepo {
   exists (callback) {
     this.version.exists(callback)
   }
+
+  /**
+   * Get repo status.
+   *
+   * @param {function(Error, Object)} callback
+   * @return {void}
+   */
+  stat (callback) {
+    waterfall([
```

Comment:
why not do the stats in parallel, and aggregate them to an object at the end?

Reply:
Good idea!

Reply:
Although there are some that use the same variable. Wouldn't that cause some problem eventually?

Reply:
If you require some of them to be series, you can still make the all series, but perhaps use `async/series` with the object argument to make this less explicit.